debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java

```diff
@@ -277,7 +303,8 @@ public final class EmbeddedEngine implements DebeziumEngine<SourceRecord> {
             OFFSET_FLUSH_INTERVAL_MS, OFFSET_COMMIT_TIMEOUT_MS,
             ERRORS_MAX_RETRIES, ERRORS_RETRY_DELAY_INITIAL_MS, ERRORS_RETRY_DELAY_MAX_MS,
             // JDBC OFFSET STORAGE FIELDS
-            OFFSET_STORAGE_JDBC_URL, OFFSET_STORAGE_JDBC_USER, OFFSET_STORAGE_JDBC_PASSWORD, OFFSET_STORAGE_JDBC_TABLE_NAME);
+            OFFSET_STORAGE_JDBC_URL, OFFSET_STORAGE_JDBC_USER, OFFSET_STORAGE_JDBC_PASSWORD, OFFSET_STORAGE_JDBC_TABLE_NAME,
+            OFFSET_STORAGE_TABLE_DDL, OFFSET_STORAGE_TABLE_SELECT);
```

Comment:
Please don't add the fields to the config. See `RedisOffsetBackingStore`

Reply:
Hi @jpechane , this was added here for the `AbstractConnectorTest`, I also checked with Chris Cranford, its duplicated unfortunately, I checked `RedisOffsetBackingStore`, I dont see an Integration test, the part I can do is to separate the config to a separate file, so its cleaner.